examples/tree_2d_dgsem/elixir_euler_kelvin_helmholtz_instability_amr.jl

```diff
@@ -21,13 +21,14 @@ function initial_condition_kelvin_helmholtz_instability(x, t,
     # change discontinuity to tanh
     # typical resolution 128^2, 256^2
     # domain size is [-1,+1]^2
+    RealT = eltype(x)
     slope = 15
-    amplitude = 0.02
-    B = tanh(slope * x[2] + 7.5) - tanh(slope * x[2] - 7.5)
-    rho = 0.5 + 0.75 * B
-    v1 = 0.5 * (B - 1)
-    v2 = 0.1 * sin(2 * pi * x[1])
-    p = 1.0
+    amplitude = convert(RealT, 0.02)
```

Comment:
Same here - do we really need this variable?

Reply:
We can remove it if it's never used.